### PR TITLE
Add namespace on flask.g

### PR DESCRIPTION
**Pull request by mortaliorchard**
When we use threads, we want to be able to pass down the values of
ows from the main thread to the child thread. Unfortunately `g` does
not work across threads, so adding an object we can attach properties
to, will work the same.

@rantonmattei @someboredkiddo

### CODE REVIEW

**Comment by rantonmattei** [comment]
The code looks right, but I'm having trouble to figure out how this will be used in a context of multi threads... probably because of my lack of knowledge with single threaded vs. multi threaded in Python.
Can you give a quick example or code sample?

**Comment by mortaliorchard** [comment]
It will make it easier everything that relates to `ows` to be pickled and rehydrate `flask.g`:

```
def test(ows):
    with app.app_context():
        g.ows = ows

Thread(target=test, args=(g.ows))
```

Where as if you don't do this, we'll have to pickle every single value on `flask.g`:
* log
* requests
* correlation_id
* owsrequest_counter
and any upcoming ones.

File: owslogger/flask_logger.py [hunk 1]
```diff
@@ -53,6 +62,18 @@ def setup(
     app.after_request(add_correlation_id_to_response)
 
 
+def get_ows():
+    """Get the ows object on the global.
+
+    This is very useful when we want to keep a copy of all the different ows
+    specific attributes set on flask.g.
+    """
```
Comment (rantonmattei) [inline owslogger/flask_logger.py]: Missing `Returns:`

**Comment by rantonmattei** [comment]
Makes more sense to me now. Thanks!
S/R+ pending the `Returns:` missing.

**Comment by mortaliorchard** [comment]
Feedback addressed before merge.

**Pull request merged**